cmd/template/cluster/flag.go

```diff
@@ -130,7 +130,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 
 	// OpenStack App only.
 	cmd.Flags().StringVar(&f.ClusterApp.ClusterUserConfigMap, flagClusterUserConfigMap, "", "Path to the user values configmap YAML file for Cluster App (OpenStack App CR only).")
-	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.1.0", "Cluster App version to be installed. (OpenStack App CR only).")
+	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.2.1", "Cluster App version to be installed. (OpenStack App CR only).")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.2.2", "Cluster App version to be installed. (OpenStack App CR only).")
```

Reply:
If 0.2.2 is the right version, please update the PR title and description.

Reply:
Thanks, yes, I was holding off as I was still deciding what the correct default version should be. I will update the title and description if I change it.